cmd/thanos/receive.go

```diff
@@ -195,17 +195,89 @@ func runReceive(
 		return errors.Wrap(err, "parse relabel configuration")
 	}
 
-	dbs := receive.NewMultiTSDB(
-		conf.dataDir,
-		logger,
-		reg,
-		tsdbOpts,
-		lset,
-		conf.tenantLabelName,
-		bkt,
-		conf.allowOutOfOrderUpload,
-		hashFunc,
+	// Note: the hashring configuration watcher
```

Comment:
I think we should revert this change, it should not be needed now that we are calling `dbs.SetHashringConfig`

Reply:
You mean we don't need `hashringConfigs` in `MultiTSDB` initialization? Calling `dbs.SetHashringConfig` in `updates` is enough?

Reply:
After reading the `receive` logs when checking locally, I think calling `dbs.SetHashringConfig` in `updates` is sufficient too. But the `updates` handling (changes handling) part is still a little unclear to me. So maybe I'll revert this `MultiTSDB` initialization later when we get changes handling work?

Reply:
Handling changes seems to be working now, so I reverted this change.